tests/CatalogTests/TestData/CatalogTestData.cs

```diff
@@ -132,7 +133,7 @@ internal static class CatalogTestData
   ""authors"": ""James Newton-King"",
   ""catalog:commitId"": ""{1}"",
   ""catalog:commitTimeStamp"": ""{2}"",
-  ""created"": ""{3}"",
+  ""created"": ""{3}"",{6}
```

Comment:
why here? Perhaps under {5}? :)

Reply:
The deprecation info gets put here due because JSON-LD alphabetizes the fields.

It was either I do this or rearrange the existing format parameters. I thought this was a smaller additive change.